src/org/labkey/test/util/AbstractContainerHelper.java

```diff
@@ -197,6 +199,11 @@ public void enableModules(List<String> moduleNames)
             }
             catch (NoSuchElementException missingModule)
             {
+                if (TestProperties.isIgnoreMissingModules())
```

Comment:
Like the previous comment I think this is backwards. Shouldn't the code throw (if the module is not there) only if you don't want to ignore missing modules?

Reply:
Nah, AssumptionViolatedException is kind of a special case when it comes to throwing it in the test.  The intent is to mute the test (not signal failure) if we are ignoring missing modules.

Reply:
The test will either fail (because we don't tell it we don't care about missing modules) or be ignored because we told it we don't care that the module is there.
The 'ignore but only when we say so' path is intended.